### PR TITLE
task(tooling): add VNet peering between bo servicebus and front office

### PR DESCRIPTION
**Pull request by m-juckes-pins**
## Describe your changes

Creating a VNet peering between back office appeals and front office for service bus

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: infrastructure/variables.tf [hunk 1]
```diff
@@ -62,6 +62,14 @@ variable "common_config" {
   })
 }
 
+variable "common_infra_config" {
+  description = "Config for the common infra"
+  type = object({
+    common_infra_vnet_name = string
+    common_infra_rg        = string
```
Comment (beejjacobs) [inline infrastructure/variables.tf:69]: The config files have `vnet_name` and `network_rg` - so we need to make sure they all match.

File: infrastructure/networking.tf [hunk 2]
```diff
@@ -49,6 +49,22 @@ resource "azurerm_virtual_network_peering" "tooling_to_bo" {
...
 
+resource "azurerm_virtual_network_peering" "bo_to_front_office" {
+  name                      = "${local.org}-peer-${local.service_name}-to-front_office-${var.environment}"
+  remote_virtual_network_id = data.azurerm_virtual_network.front_office_vnet.id
+  resource_group_name       = azurerm_virtual_network.main.resource_group_name
+  virtual_network_name      = azurerm_virtual_network.main.name
+}
+
+resource "azurerm_virtual_network_peering" "front_office_to_bo" {
+  name                      = "${local.org}-peer-tooling-to-${local.service_name}-${var.environment}"
```
Comment (beejjacobs) [inline infrastructure/networking.tf:60]: We should replace tooling here, else it'll be the same name as the tooling peering.

`tooling` -> `front-office` to be consistent with the other one?

File: infrastructure/networking.tf [hunk 3]
```diff
@@ -49,6 +49,22 @@ resource "azurerm_virtual_network_peering" "tooling_to_bo" {
   provider = azurerm.tooling
 }
 
+resource "azurerm_virtual_network_peering" "bo_to_front_office" {
+  name                      = "${local.org}-peer-${local.service_name}-to-front_office-${var.environment}"
```
Comment (beejjacobs) [inline infrastructure/networking.tf:53]: For consistency, be great to change the _ to a -

**Review by beejjacobs — CHANGES_REQUESTED** [review]
Looks good! Just a couple of comments. Might be worth running a plan against dev to check it's all OK.

Also probably be nice to squash this down to one commit?

File: infrastructure/networking.tf (hunk 3 above)
Comment (m-juckes-pins) [reply infrastructure/networking.tf:53]: good spot!

**Review by beejjacobs — DISMISSED** [review]
Comments addressed, looks good!

**Review by beejjacobs — APPROVED** [review]

**Review by beejjacobs — APPROVED** [review]

**Pull request merged**